assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -50,6 +52,7 @@ const resultsTableConfig = {
           >
             {checkID}
           </span>
+          <span>{premium ? <PremiumPill /> : ' '}</span>
```

Comment:
Cannot you simply put `{premium && <PremiumPill />}` without the span?
I mean, the pill is already a span. Or do you need yes or yes an empty span for styling?
It looks ugly hehe

Reply:
Yeah you are fully right, the span can go away :D